server/src/main/java/org/spine3/server/validate/FloatFieldValidator.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.validate;
+
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Any;
+import com.google.protobuf.Descriptors.FieldDescriptor;
+
+import static java.lang.Math.abs;
+import static org.spine3.protobuf.Messages.newFloatValue;
+import static org.spine3.protobuf.Messages.toAny;
+
+/**
+ * Validates fields of {@link Float} types.
+ *
+ * @author Alexander Litus
+ */
+/* package */ class FloatFieldValidator extends FloatFieldValidatorBase<Float> {
+
+    /**
+     * Creates a new validator instance.
+     *
+     * @param descriptor a descriptor of the field to validate
+     * @param fieldValues field values to validate
+     */
+    /* package */ FloatFieldValidator(FieldDescriptor descriptor, ImmutableList<Float> fieldValues) {
+        super(descriptor, fieldValues);
+    }
+
+    @Override
+    protected Float toNumber(String value) {
+        final Float min = Float.valueOf(value);
+        return min;
+    }
+
+    @Override
+    protected Float getAbs(Float value) {
+        final Float abs = abs(value);
+        return abs;
+    }
+
+    @Override
+    protected Any wrapToMsgAndAny(Float value) {
```

Comment:
A better name would be simply `wrap()` with proper documentation. 

It is assumed that plain Java type cannot be put into `Any` without being converted into proper Protobuf instance first. In that sense, we don't wrap the passed value into the message, but convert.